libs/constants/index.js

```diff
@@ -76,9 +76,9 @@ const scriptSrcValues = [
    * we must always update the CSP hash (see instructions there).
    */
   // - Previous hash (to avoid cache invalidation issues):
-  "'sha256-GA8+DpFnqAM/vwERTpb5zyLUaN5KnOhctfTsqWfhaUA='",
-  // - Current hash:
   "'sha256-R1d+R+j7z1re3EpSIuqRpflohOvTm3fAxKvfEiXgC1o='",
```

Comment:
Keep previous hash, as it already is the previous hash (from last night's build):

```suggestion
  "'sha256-GA8+DpFnqAM/vwERTpb5zyLUaN5KnOhctfTsqWfhaUA='",
```